docs/source/specs/openapi.json

```diff
@@ -215,6 +215,21 @@
               "type": "string"
             }
           },
+          {
+            "name": "uuid",
+            "in": "query",
+            "description": "A list of UUIDs to filter listed groups.",
+            "required": false,
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "string"
+              },
+              "collectionFormat": "csv"
```

Comment:
It looks like this is a 2.0 thing, not compatible with 3.0? https://swagger.io/docs/specification/describing-parameters/

Reply:
Oh, yep. Dang. I had hopes it'd help clarify, but I'll pull it out